src/metrics.js

```diff
@@ -1,25 +1,32 @@
 const Prometheus = require('prom-client');
 
-const requestCount = new Prometheus.Counter({
-  name: 'http_requests_total',
-  help: 'Counter for total requests received',
-  labelNames: ['route', 'method', 'status'],
-});
+/**
+ * @param prefix - metrics name prefix
+ * request counter
+ */
+function requestCountGenerator(prefix = '') {
+  return new Prometheus.Counter({
+    name: `${prefix}http_requests_total`,
+    help: 'Counter for total requests received',
+    labelNames: ['route', 'method', 'status'],
+  });
+}
 
 /**
  * @param {!Array} buckets - array of numbers, representing the buckets for
+ * @param prefix - metrics name prefix
  * request duration
  */
-function requestDurationGenerator(buckets) {
+function requestDurationGenerator (buckets, prefix = '') {
```

Comment:
also theres an extra space between the function name and the `(`

I've added a linting script to the project `npm run lint`